### PR TITLE
Update default wallet name 

### LINKED ISSUE #17951: Wallet: Properly set name on default wallet account
Currently the default wallet account is named "Ethereum Account", however in the redesign this should be "Account 1" 

Update this in Status Go to the correct details

### PR DESCRIPTION
**Pull request by jo-mut**
Updates the defaul wallet name to `Account 1`

fixes https://github.com/status-im/status-mobile/issues/17951

### Summary
In the current Ui the default wallet name is `Ethereum account`. This needs to be updated to `Account 1` as is on the designs

https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1-46718&mode=design&t=r7mG6i8ADZHOPvXS-4

**Before**

**After**

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]

**Comment by pavloburykh** [comment]
@jo-mut thanx for the PR. Please, do not forget to add status-go label to PRs containing changes on go side. Thank you.

**Comment by pavloburykh** [comment]
@jo-mut couple more things about the PRs containing go: 

1. Before sending PR into testing, please make sure that corresponding go PR is reviewed, approved, rebased to the latest develop and does not contain conflicts. In this case correspondent go PR https://github.com/status-im/status-go/pull/4566 is not approved, does not contain latest develop changes and contains bunch of conflicts. 

2. Whenever you rebase the PR containing go changes, please **do not forget** to rebase BOTH mobile and correspondent go branch. In such cases keeping go branch updated can be even more important than keeping updated the mobile branch (although both are important). If you do not keep go branch updated it may lead to some new serious issues (even not related to your PR) leak into our develop once you will rebase your go branch before merging and bumping go version in mobile develop. 

So, currently PR is blocked from testing until go [PR](https://github.com/status-im/status-go/pull/4566) is approved, rebased and has all the conflicts resolved. 

Feel free to ping me up when PR is ready. Thank you.

File: 1 [hunk 1]
```diff
@@ -0,0 +1,16 @@
+add general qr code scan screen
```
Comment (J-Son89) [inline 1]: we can remove this :)

**Review by J-Son89 — APPROVED** [review]

**Comment by jo-mut** [comment]
@pavloburykh you can proceed with this now. status-go pr is merged and this pr works as expected

**Comment by pavloburykh** [comment]
> @pavloburykh you can proceed with this now. status-go pr is merged and this pr works as expected

hi @jo-mut. Could you please clarify why has go PR been merged prior to testing mobile PR?

**Comment by jo-mut** [comment]
oooh @pavloburykh I totally have no excuse for this. I am totally at fault here. Please advise me on what I can do moving forward with this PR.

**Comment by pavloburykh** [comment]
> oooh @pavloburykh I totally have no excuse for this. I am totally at fault here. Please advise me on what I can do moving forward with this PR.

We will test like it is. Hope there will be no bugs as there is only 1 string of code change . Just next time make sure related mobile PR is tested before merging the go PR. Thank you.

**Comment by jo-mut** [comment]
my bad. I will addd this note to my personal QA process notebook just not to miss the steps. thank you

**Comment by VolodLytvynenko** [comment]
@jo-mut Thank you for PR. No issues from my side. PR can be merged

**Pull request merged**